### ISSUE TITLE
Desk not showing after pasting google analytics script in HTML HEADER & ROBOTS section of website setting

### ISSUE THREAD

**Issue by erpwala** [frappe/erpnext#14850]
n", "hide_footer_signup": 0, "home_page": "homepage", "idx": "0", "modified": "2018-07-09 15:10:24.440345", "modified_by": "jawahar.mallah@netmanthan.com", "name": "Website Settings", "navbar_search": 0, "owner": "Administrator", "top_bar_items": [ { "creation": "2018-07-08 14:55:12.344095", "docstatus": 0, "doctype": "Top Bar Item", "idx": 1, "label": "Contact", "modified": "2018-07-09 15:10:24.440345", "modified_by": "jawahar.mallah@netmanthan.com", "name": "bb5425dc65", "owner": "Administrator", "parent": "Website Settings", "parentfield": "top_bar_items", "parenttype": "Website Settings", "right": 0, "url": "/contact" }, { "creation": "2018-07-08 14:55:12.344095", "docstatus": 0, "doctype": "Top Bar Item", "idx": 2, "label": "Blog", "modified": "2018-07-09 15:10:24.440345", "modified_by": "jawahar.mallah@netmanthan.com", "name": "604597e701", "owner": "Administrator", "parent": "Website Settings", "parentfield": "top_bar_items", "parenttype": "Website Settings", "right": 0, "url": "/blog" }, { "creation": "2018-07-08 14:55:12.344095", "docstatus": 0, "doctype": "Top Bar Item", "idx": 3, "label": "Products", "modified": "2018-07-09 15:10:24.440345", "modified_by": "jawahar.mallah@netmanthan.com", "name": "c76755e82b", "owner": "Administrator", "parent": "Website Settings", "parentfield": "top_bar_items", "parenttype": "Website Settings", "right": 0, "url": "/products" } ], "website_theme": "Standard" } }; frappe.csrf_token = "cfeda755a65c89b5040e73814911ce3d4194a3d63832dbc189395fc2";

 **_**Desk not showing after pasting google analytics script in HTML HEADER & ROBOTS section of website setting showing above error**_** (Desk Not showing after successful login,  its shows above message)
### **below  is the code what i pasted.**

<script async src="https://www.googletagmanager.com/gtag/js?id=UA-121949883-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-121949883-1');
</script>

**Comment by jigneshpshah** [comment]
Same Issue .  Desk not showing after pasting google analytics script in HTML HEADER & ROBOTS section of website setting.


**Tested on ERPNext Cloud
ERPNext: v10.1.46 (master)
Frappe Framework: v10.1.45 (master)**

If we enter just the Analytics ID in Website > Integration > 
![image](https://user-images.githubusercontent.com/4900815/44082089-f20c980c-9fcd-11e8-8860-c16d5163bc73.png)

and then if we check the source of the webpage, the analytics script code is missing.

Any solution to use Google Analytics on website. As this is a critical feature.

**Comment by generare** [comment]
Still having this issue?

**Comment by jaichavan** [comment]
Hello, @jigneshpshah does the issue persist?

**Comment by jigneshpshah** [comment]
Checked on 
ERPNext: v11.0.3-beta.34 () (staging)
Frappe Framework: v11.0.3-beta.48 () (staging)

Desk not showing after pasting google analytics script in HTML HEADER & ROBOTS section of website setting.

**Comment by parsimonyERP** [comment]
Is this a solution?: https://discuss.erpnext.com/t/urgent-help-please/24609

**Comment by jigneshpshah** [comment]
No. That is just how to erase the script code from the database table instead of website settings doctype.

But it seems , there is no solution of how we can use google analytics code.
There are two options 
1) to put the google analytics id in the website settings > Integration section. 
After putting it , when we see any webpage > view source , we don't find the google script code

2) is to put the google script code directly to the Website  Settings > HTML Header <head> section.
After putting it, we are able to see the google script code in any web page, but we are not able to access the desk.

**Comment by zerodiscount** [comment]
Seems tracking is working with just using the Google Analytics ID (without putting the script under <head>)?? I just tried with the ID and google analytics console is showing the correct number of users in real time.... (including mobile versus desktop).

**Comment by jigneshpshah** [comment]
Great..Could you please specify the version on which you tested..

**Comment by zerodiscount** [comment]
v11.1.14.  Using the script though does cause the above error. Also do not know if all analytics are being captured correctly without the script...

**Comment by garyritchie** [comment]
This just happened to me, running v12.3.9.

What's the process to undo the GA script addition so I can access Desk?

**Comment by jigneshpshah** [comment]
Just go to the database and update tabsingles table and update the head field value of website settings.

**Comment by garyritchie** [comment]
To confirm, there's no Google Analytics tracking code in my site, despite putting the code into Website Settings --> INTEGRATIONS --> Google Analytics ID.

Hoping there's a fix or workaround soon.

**Comment by 1zaak** [comment]
Not working for me as well.

> ...there's no Google Analytics tracking code in my site, despite putting the code into Website Settings --> INTEGRATIONS --> Google Analytics ID.

**Comment by ImtiyazAM** [comment]
Hi Everyone.  I was trying to integrate google analytics with erpnext by adding google analytic measurment ID, But its not connecting.  in the google analytic dashboard its showing like this
"Data collection isn’t active for your website. If you installed tags more than 48 hours ago, make sure they are set up correctly".
Do i need to add Google tag manually anywhere in the erpnext page?

**Comment by khab1** [comment]
Did you manage to get it working? @ImtiyazAM . I have the same issue on v15.37.0

**Comment by federicocalvo** [comment]
Is the issue still active today?

**Comment by rmehta** [comment]
The future of website is https://github.com/frappe/builder

**Comment by federicocalvo** [comment]
@rmehta builder generates mixed feelings, and as always, you can't argue with the priority of the tasks or you get a (do it yourself), it's clear. I say this because they are clearly prioritizing issues that do not affect the majority.

But having some experience in webs, knowing html and css, it still has a very big curve.

For there to be a massive adaptation (and that would add to frappe seeing that there are thousands of websites made in builders or passed to builders) should prioritizing the number of sections and start with templates, since the vast majority, use this in the current website.

**Comment by rmehta** [comment]
Forgot to add there is already a google analytics ID property in the Website Settings

**Comment by federicocalvo** [comment]
> Forgot to add there is already a google analytics ID property in the Website Settings

Hi, does not work, there are several posts about it both on discuss and GitHub
